functions/go/starlark/README.md

```diff
@@ -4,27 +4,36 @@
 
 <!--mdtogo:Short-->
 
-Run a Starlark script to mutate or validate resources.
+The `starlark` function contains a Starlark interpreter to run a Starlark script
+to mutate or validate resources.
+
+It follows the [executable configuration] pattern. It makes writing simple
+function trivial.
```

Comment:
I'd avoid general statements like making things "trivial". There are friction points in executable config flow.

Reply:
I updated the wording here.

> There are friction points in executable config flow.

Is there anything we can address in this PR?